ocean_lib/ocean/ocean.py

```diff
@@ -216,11 +219,57 @@ def get_user_orders(
 
         return _orders
 
+    @property
+    @enforce_types
+    def fixed_rate_exchange(self):
+        return FixedRateExchange(
+            self.web3, get_address_of_type(self.config, "FixedPrice")
+        )
+
+    @enforce_types
+    def create_fixed_rate(
+        self,
+        erc20_token: ERC20Token,
+        base_token: ERC20Token,
+        amount: int,
+        from_wallet: Wallet,
+    ) -> bytes:
+        fixed_price_address = get_address_of_type(self.config, "FixedPrice")
+        erc20_token.approve(fixed_price_address, amount, from_wallet)
+        addresses = [
+            self.OCEAN_address,
```

Comment:
I think it has to be the base_token.address. It works in this case because you tested with base_token == OCEAN token, but the general case should be different, if I'm not mistaken